mfa-fraud (#232)
GovTechSG/techpass-user-docs: docs/mfa-fraud-alert.md

If a user encounters an unexpected Number Matching challenge during login, the user should select the **NO, IT'S NOT ME** option and click on the **Report** button on the subsequent screen. This action will place the user's account in the **Risky Users with High Risk Level** state and trigger an investigation by the security team.

![not-me](/docs/assets/images/mfa-fraud/unexpected-no.png)

Upon selecting the **NO, IT'S NOT ME** option and reporting the incident, the security team will investigate the situation.

If the malicious user or the actual account holder attempts to log in after entering the correct credentials, a warning message will be displayed. 

![not-me](/docs/assets/images/mfa-fraud/acc-risk.png)

The user can click the **Verify** button and the user will be provided options for the user to verify their identity. However, none of the verification steps will be successful, as the following screen will consistently appear:

![not-me](/docs/assets/images/mfa-fraud/request-denied.png)

This serves as a security measure to prevent unauthorized access and ensure the protection of user accounts.

> An email will be sent to the user's registered email address to facilitate communication and resolution. The user should check their email inbox for this notification.

Reviewer: ```suggestion
A remediation email will be sent to the user's registered email address. The user should check their email inbox for this notification.
```

remediation feels like a fitting word compared to resolution. so for your consideration ))
